lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,24 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
-      origin_branch = current_branch
-      run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
-      run_command_with_label "git checkout #{origin_branch}"
+    def update_destination(options = {})
+      feature_branch = options[:feature_branch]
+      base_branch = options[:base_branch]
+      remote = options[:remote] || 'origin'
+      run_command_with_label "git checkout #{base_branch}"
+      run_command_with_label "git pull #{remote} #{base_branch}"
+      run_command_with_label "git checkout #{feature_branch}"
+    end
+
+    def update_feature_branch(options = {})
```

Comment:
I believe you mentioned that we should update the base branch in your sample code: https://github.com/reenhanced/gitreflow/pull/164#discussion-diff-59970260

with: `update_destination(base_branch, remote: remote)` 

The point in `git reflow refresh` is to keep our local feature branch in sync with the local base branch which should be in sync with the remote base branch. `git pull origin feature-branch && git pull remote base-branch` only keeps the local feature branch in sync with the remote base + feature branches.  In order to do the behavior I described and update our local base branch, we should checkout to to the base first?